pkg/oci/repository.go

```diff
@@ -38,53 +41,46 @@ type options struct {
 	insecure   bool
 }
 
-// WithAuthFromSecret returns an option that configures the repository to use the provided keychain.
```

Comment:
you removed the `auth` options because we don't need them at the moment? They were in case you want to fetch from a remote registry and you need to provide `auth` information.

Reply:
The OCI registry itself will never fetch from a remote registry. The registry is always in-cluster I believe.

The fetching from the outside is always done via OCM. Now. That said. I do think that someone might want to set up a secured in-cluster registry maybe, but since WE are controlling the registry we probably can keep it at this level. Since it's in-cluster and not exposed in any way.

I can put it back, but this will always be in git's history if we ever need it. :)